`npm run build` fails

The following command fails in the current version (0.2.1):

```
$ npm run build

> @ga4gh/gh-openapi-docs@0.2.1 build /home/tschaffter/persist/dev/gh-openapi-docs
> npx webpack-cli --mode=development

Hash: 1d8f19947d661805b1b3
Version: webpack 4.42.0
Time: 304ms
Built at: 08/29/2020 9:42:20 PM
    Asset      Size           Chunks             Chunk Names
bundle.js  8.33 KiB  gh-openapi-docs  [emitted]  gh-openapi-docs
Entrypoint gh-openapi-docs = bundle.js
[./src/gh-openapi-docs.js] 4.44 KiB {gh-openapi-docs} [built] [failed] [1 error]

ERROR in ./src/gh-openapi-docs.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module '@babel/compat-data/corejs3-shipped-proposals'
Require stack:
- /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/@babel/preset-env/lib/polyfills/corejs3/usage-plugin.js
- /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/@babel/preset-env/lib/index.js
- /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/@babel/core/lib/config/files/plugins.js
- /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/@babel/core/lib/config/files/index.js
- /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/@babel/core/lib/index.js
- /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/babel-loader/lib/index.js
- /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/loader-runner/lib/loadLoader.js
- /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/loader-runner/lib/LoaderRunner.js
- /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/webpack/lib/NormalModule.js
- /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/webpack/lib/NormalModuleFactory.js
- /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/webpack/lib/Compiler.js
- /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/webpack/lib/webpack.js
- /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/webpack-cli/bin/utils/validate-options.js
- /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/webpack-cli/bin/utils/convert-argv.js
- /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/webpack-cli/bin/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
    at Function.Module._load (internal/modules/cjs/loader.js:841:27)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (/home/tschaffter/persist/dev/gh-openapi-docs/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/home/tschaffter/persist/dev/gh-openapi-docs/node_modules/@babel/preset-env/lib/polyfills/corejs3/usage-plugin.js:10:55)
    at Module._compile (/home/tschaffter/persist/dev/gh-openapi-docs/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (/home/tschaffter/persist/dev/gh-openapi-docs/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/home/tschaffter/persist/dev/gh-openapi-docs/node_modules/@babel/preset-env/lib/index.js:29:44)
    at Module._compile (/home/tschaffter/persist/dev/gh-openapi-docs/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (/home/tschaffter/persist/dev/gh-openapi-docs/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at requireModule (/home/tschaffter/persist/dev/gh-openapi-docs/node_modules/@babel/core/lib/config/files/plugins.js:165:12)
    at loadPreset (/home/tschaffter/persist/dev/gh-openapi-docs/node_modules/@babel/core/lib/config/files/plugins.js:83:17)
    at createDescriptor (/home/tschaffter/persist/dev/gh-openapi-docs/node_modules/@babel/core/lib/config/config-descriptors.js:154:9)
    at /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/@babel/core/lib/config/config-descriptors.js:109:50
    at Array.map (<anonymous>)
    at createDescriptors (/home/tschaffter/persist/dev/gh-openapi-docs/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
    at createPresetDescriptors (/home/tschaffter/persist/dev/gh-openapi-docs/node_modules/@babel/core/lib/config/config-descriptors.js:101:10)
    at /home/tschaffter/persist/dev/gh-openapi-docs/node_modules/@babel/core/lib/config/config-descriptors.js:58:104
    at cachedFunction (/home/tschaffter/persist/dev/gh-openapi-docs/node_modules/@babel/core/lib/config/caching.js:62:27)
    at cachedFunction.next (<anonymous>)
    at evaluateSync (/home/tschaffter/persist/dev/gh-openapi-docs/node_modules/gensync/index.js:244:28)
    at sync (/home/tschaffter/persist/dev/gh-openapi-docs/node_modules/gensync/index.js:84:14)
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! @ga4gh/gh-openapi-docs@0.2.1 build: `npx webpack-cli --mode=development`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the @ga4gh/gh-openapi-docs@0.2.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/tschaffter/.npm/_logs/2020-08-29T21_42_20_906Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -38,6 +38,7 @@
   },
   "devDependencies": {
     "@babel/cli": "^7.10.1",
+    "@babel/compat-data": "^7.10.1",
     "@babel/core": "^7.10.2",
     "@babel/node": "^7.10.1",
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",

--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
     "shelljs": "^0.7.0",
     "swagger-ui-dist": "3.24.0",
     "update-notifier": "3.0.1",
-    "yargs-parser": "15.0.0"
+    "yargs-parser": "^19.0.4"
   },
   "devDependencies": {
     "@babel/cli": "^7.10.1",

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -40,5 +40,9 @@ module.exports = {
     },
     plugins: [
         new webpack.BannerPlugin({ banner: "#!/usr/bin/env node", raw: true })
-    ]
+    ],
+    stats: {
+        // Ignore warnings due to yarg's dynamic module loading
+        warningsFilter: [/node_modules\/yargs/]
+    }
 }

